buildtest/cli/report.py

```diff
@@ -590,10 +590,11 @@ def print_report(
                 t = t[:count]
 
             if not noheader:
-                print("|".join(self.display_table.keys()))
+                console.print("|".join(self.display_table.keys()), style=consoleColor)
 
             for i in t:
-                print("|".join(i))
+                join_string = "|".join(i)
```

Comment:
rename this variable `join_string` perhaps `line` or `row_entry` or something else